### PR TITLE
CLOUDP-18701 (Mark) Add security policy API support

### PR DESCRIPTION
**Pull request by dgolub**

### CODE REVIEW

File: policy.go [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+	defer wrappers.LsaFreeMemory((*byte)(unsafe.Pointer(rights)))
+	rightNames := make([]AccountRightName, count)
+	for i := uint32(0); i < count; i++ {
+		buf := make([]uint16, rights.Length)
+		wrappers.RtlMoveMemory(
+			(*byte)(unsafe.Pointer(&buf[0])),
+			(*byte)(unsafe.Pointer(rights.Buffer)),
+			uintptr(rights.Length))
+		rightNames[i] = AccountRightName(syscall.UTF16ToString(buf))
+		rights = (*wrappers.UNICODE_STRING)(unsafe.Pointer(uintptr(unsafe.Pointer(rights)) + unsafe.Sizeof(*rights)))
```
Comment (markbenvenuto) [inline policy.go]: Is there no way to do this loop as by accessing each array member as `rights[i]`?

File: wrappers/ntsecapi.go [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+	POLICY_SET_DEFAULT_QUOTA_LIMITS = 0x0080
+	POLICY_SET_AUDIT_REQUIREMENTS   = 0x0100
+	POLICY_AUDIT_LOG_ADMIN          = 0x0200
+	POLICY_SERVER_ADMIN             = 0x0400
+	POLICY_LOOKUP_NAMES             = 0x0800
+	POLICY_ALL_ACCESS               = STANDARD_RIGHTS_REQUIRED | POLICY_VIEW_LOCAL_INFORMATION | POLICY_VIEW_AUDIT_INFORMATION | POLICY_GET_PRIVATE_INFORMATION | POLICY_TRUST_ADMIN | POLICY_CREATE_ACCOUNT | POLICY_CREATE_SECRET | POLICY_CREATE_PRIVILEGE | POLICY_SET_DEFAULT_QUOTA_LIMITS | POLICY_SET_AUDIT_REQUIREMENTS | POLICY_AUDIT_LOG_ADMIN | POLICY_SERVER_ADMIN | POLICY_LOOKUP_NAMES
+)
+
+const (
+	SE_INTERACTIVE_LOGON_NAME             = "SeInteractiveLogonRight"
```
Comment (markbenvenuto) [inline wrappers/ntsecapi.go]: See the list in `winnt.h`. Search for 
```
#define SE_AUDIT_NAME                     TEXT("SeAuditPrivilege")
```

File: policy.go (hunk 1 above)
Comment (dgolub) [inline policy.go]: Unfortunately, there's no pointer arithmetic in Go, so I'm pretty sure that this is the best we can do.

File: wrappers/ntsecapi.go (hunk 2 above)
Comment (dgolub) [inline wrappers/ntsecapi.go]: We already define those in winnt.go.

File: policy.go (hunk 1 above)
Comment (markbenvenuto) [inline policy.go]: :disappointed:

File: wrappers/ntsecapi.go (hunk 2 above)
Comment (markbenvenuto) [inline wrappers/ntsecapi.go]: Now I see they are separate lists with no duplication.

**Pull request merged**